uwrt_mars_rover_arm/uwrt_mars_rover_arm_description/uwrt_mars_rover_arm_description/urdf/arm_params.yaml

```diff
@@ -0,0 +1,45 @@
+dimensions:
+  box_size: 1
+
+  turntable_radius: 0.6
+  turntable_length: 0.1
+  
+  upper_arm_radius: 0.1
+  upper_arm_length: 1
+
+  forearm_radius: 0.1
+  forearm_length: 0.5
+
+  forearm_roll_radius: 0.1
+  forearm_roll_length: 0.5
+
+  wrist_radius: 0.1
+  wrist_length: 0.5
+
+  claw_width: 0.05
+  claw_length: 0.25
+
+inertia:
+  default_mass: 1
+
+joints: 
+  default_limit_effort: 10
+  default_limit_vel: 0.5
+
+  shoulder_lower: !degrees  -90.0
```

Comment:
is the !degrees something in yaml?

Reply:
its just to input degrees instead of radians for the angles, should it be changed to radians?

Reply:
Oh I didn't know the parser recognized these tags. Cool! How did u find out about them?

Reply:
the first part of this, http://wiki.ros.org/rosparam

Reply:
Neat! I also didn't know python math expressions were parsed in ros yamls!